modules/Quiz/resources/IVQModule.js

```diff
@@ -53,11 +53,28 @@
                     }
                     else {
                         $.quizParams = data[1];
+                        var banSeekTrigger = false;
+                        var dataForBanSeek = {
+                            status:false,
+                            alertText:''
+                        };
                         $.grep($.quizParams.uiAttributes, function (e) {
+                            
+                            if (e.key == "banSeek" && e.value) {
+                                banSeekTrigger = true;
+                                dataForBanSeek.status = true;
```

Comment:
@MaksymKorniiov  - I see that a few lines below there is a e.value.toLowerCase() 
Do you see a reason for use/not use the same protection here?